.eslintrc

```diff
@@ -0,0 +1,73 @@
+{
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "extends": [
+    "eslint:recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
+    "plugin:react-hooks/recommended",
+    "plugin:jsx-a11y/strict",
+    "plugin:jest/recommended",
+    "prettier"
+  ],
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "rules": {
+    "@typescript-eslint/no-explicit-any": 1,
+    "@typescript-eslint/no-inferrable-types": [
+      "warn",
+      {
+        "ignoreParameters": true
+      }
+    ],
+    "@typescript-eslint/no-unused-vars": "error",
```

Comment:
Wouldn't warning be enough for no-unused-vars? 

It's annoying during development when you define a var somewhere and it doesn't compile.

Reply:
I have removed the `nounusedlocals` from `tsconfig` for the same reason, this will only show up in editors and when committing (which also can be skipped in WIP Merge Requests).
Not having it in `tsconfig` would make it compile anyway.